server/storages/clickhouse.go

```diff
@@ -284,3 +284,7 @@ func (ch *ClickHouse) getAdapters() (*adapters.ClickHouse, *TableHelper) {
 	num := rand.Intn(len(ch.adapters))
 	return ch.adapters[num], ch.chTableHelpers[num]
 }
+
+func (ch *ClickHouse) TestBatchProcessing(testName string, events []map[string]interface{}) error {
```

Comment:
TestBatchProcessing can't return not supported error if destination supports batch mode.

Imagine the situation: User have created ClickHouse destination in batch mode and wants to test connection. Incomprehensible error will be returned, and it is a bug.

E.g. we could run stream test connection pipeline (as they are identical)

Reply:
Refactoring made `batch` testing available for `ClickHouse` storage type.